synapse/handlers/typing.py

```diff
@@ -362,11 +362,13 @@ async def _recv_edu(self, origin: str, content: JsonDict) -> None:
             )
             return
 
-        domains = await self._storage_controllers.state.get_current_hosts_in_room(
+        # Let's check that the origin server is in the room before accepting the typing event.
+        # We don't want to block waiting on a partial state so take the approximation if needed.
```

Comment:
nit: can we wrap comments at 88 columns (same as the line length in the linter config)?

```suggestion
        # Let's check that the origin server is in the room before accepting the typing
        # event. We don't want to block waiting on a partial state so take an
        # approximation if needed.
```